.circleci/config.yml

```diff
@@ -1,31 +1,34 @@
 version: 2
+commands:
 jobs:
-  integration:
-    machine:
-      image: circleci/classic:latest
+  go-lint:
+    docker:
+      - image: circleci/golang:1.11
     steps:
       - checkout
-      - run: "./integrationTest.sh"
+      - run: "! go mod tidy -v 2>&1 | read"
+      - run: go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.25.0
+      - run: "golangci-lint run"
+  py-lint:
+    docker:
+      - image: circleci/python:3.6.8
+    steps:
+      - run: sudo pip3 install isort flake8
+      - run: "isort -rc . --check-only"
+      - run: "flake8 . --max-line-length=88"
+      - run: "black . --check"
   unit:
     docker:
       - image: circleci/golang:1.11
     steps:
       - checkout
       - run: go test ./...
-  lint:
-    docker:
-      - image: circleci/golang:1.11
+  integration:
+    machine:
+      image: circleci/classic:latest
     steps:
       - checkout
-      - run: "! go fmt ./... 2>&1 | read"
-      - run: "! go vet ./... 2>&1 | read"
```

Comment:
for all the other lines i removed: golangci-lint covers this, and the existing line (with the `! ....... | read`) wasn't doing anything